x-pack/qa/smoke-test-watcher-with-security/src/test/java/org/elasticsearch/smoketest/SmokeTestWatcherWithSecurityIT.java

```diff
@@ -330,6 +331,10 @@ private ObjectPath getWatchHistoryEntry(String watchId, String state) throws Exc
                 String watchid = objectPath.evaluate("hits.hits.0._source.watch_id");
                 assertThat(watchid, is(watchId));
                 objectPathReference.set(objectPath);
+            } catch (ResponseException e) {
+                final String err = "Failed to perform search of watcher history - " + e;
+                logger.info(err);
```

Comment:
Any particular reason not to use

```suggestion
                logger.info("Failed to perform search of watcher history", e);
```

Here instead so that you get the whole stacktrace?

Reply:
thanks @dakrone , I updated this and the other code (from which this came) to preserve the stack trace. https://github.com/elastic/elasticsearch/pull/43781/commits/e448050afabf469ae422ebdd11920a163f0c0573